internal/xds/translator/listener.go

```diff
@@ -30,7 +37,8 @@ func buildXdsListener(httpListener *ir.HTTPListener) (*listener.Listener, error)
 		},
 		// Use only router.
 		HttpFilters: []*hcm.HttpFilter{{
-			Name: wellknown.Router,
+			Name:       wellknown.Router,
```

Comment:
in fact name can be omit because it isn't needed

Reply:
yes, prefer to keep it for better reading